xml/ha_storage_protection.xml

```diff
@@ -467,347 +617,254 @@ Timeout (allocate) : 2
+  <para>
+   As a final step, test the SBD devices as described in <xref
+   linkend="pro.ha.storage.protect.sbd.test" xrefstyle="select:label"/>.
+  </para>
 
-    <para>To enable the software watchdog, use the following steps:</para>
-     <procedure>
-       <step>
-         <para>Add the following line to <filename>/etc/init.d/boot.local</filename>:</para>
-         <screen>modprobe softdog</screen>
-       </step>
-       <step>
-         <para>Open the file <filename>/etc/sysconfig/sbd</filename>:</para>
-         <screen>SBD_DEVICE="/dev/<replaceable>SBD</replaceable>"</screen>
-         <para>In case you have multiple devices, use the semicolon as a separator.</para>
-       </step>
-       <step>
-         <para>Test the SBD daemon:</para>
-         <screen>&prompt.root;<command>sbd</command> -d /dev/<replaceable>SBD</replaceable> list
-0       &node1;      clear
-1       &node2;        clear</screen>
-         <para>The command will list all cluster nodes that have ever been
-       started against this device. Running nodes should have a
-        <literal>clear</literal> state. Fenced nodes, but not yet restarted,
-       shows the appropriate fencing message.</para>
-        <remark>toms 2018-04-05: What to do when the number of nodes doesn't
-         match the expected number of cluster nodes?</remark>
-       </step>
-       <step>
-         <para>Send a test message to one of your nodes, for example from
-          &node1; to &node2;:</para>
-         <screen>&prompt.root;<command>sbd</command>  -d /dev/<replaceable>SBD</replaceable> message &node2; test</screen>
-       </step>
-      <step>
-       <para>Check the message on &node2;:</para>
-       <screen>&prompt.root;<command>systemctl</command> status sbd
-[...]
-Sep 22 17:01:00 &node1; sbd: [13412]: info: Received command test from &node1;</screen>
-      </step>
-     </procedure>
-   </sect3>
+  <procedure xml:id="pro.ha.storage.protect.sbd.test">
+   <title>Testing the SBD Devices</title>
+    <step>
+     <para> The following command will dump the node slots and their current
+      messages from the SBD device: </para>
+     <screen>&prompt.root;<command>sbd</command> -d /dev/<replaceable>SBD</replaceable> list</screen>
+    <para> Now you should see all cluster nodes that have ever been started with SBD listed here.
+     For example, if you have a two-node cluster, the message slot should show
+      <literal>clear</literal> for both nodes:</para>
+     <screen>0       &node1;        clear
+1       &node2;          clear</screen>
+    </step>
+    <step>
+     <para> Try sending a test message to one of the nodes: </para>
+     <screen>&prompt.root;<command>sbd</command> -d /dev/<replaceable>SBD</replaceable> message &node1; test</screen>
+    </step>
+    <step>
+     <para> The node will acknowledge the receipt of the message in the system
+      log files: </para>
+     <screen>Aug 29 14:10:00 &node1; sbd: [13412]: info: Received command test from &node2;</screen>
```

Comment:
Based on https://github.com/ClusterLabs/sbd/pull/48 this will be like:

May 03 16:08:31 &node1; sbd[66139]:   /dev/<replaceable>SBD</replaceable>:   notice: servant: Received command test from &node2; on disk /dev/<replaceable>SBD</replaceable>

Reply:
Many thanks! Integrated with https://github.com/SUSE/doc-sleha/commit/facfc65fb7531f2e.